packages/language-server/package.json

```diff
@@ -34,7 +34,11 @@
   "devDependencies": {
     "@types/node-fetch": "2.5.5",
     "@ui5-language-assistant/semantic-model-types": "^1.0.0",
-    "vscode-languageserver-types": "3.15.1"
+    "vscode-languageserver-types": "3.15.1",
+    "tmp-promise": "2.0.2",
+    "@types/tmp": "0.1.0",
+    "rimraf": "3.0.2",
```

Comment:
- rimraf is already a common dev dep at the root of the repo.
- `@types/rimraf` could also be added to the root package.

Reply:
I'll move it